runelite-client/src/main/java/net/runelite/client/plugins/banktags/TagManager.java

```diff
@@ -98,6 +124,21 @@ void setTagString(int itemId, String tags, boolean variation)
 		}
 	}
 
+	public void setTagItemQuantity(String tag, int itemId, int amount, boolean variation)
+	{
+		itemId = getItemId(itemId, variation);
+
+		if (amount <= 1)
```

Comment:
This should be keying on empty string or null. Requiring someone to enter -1 seems wrong